openedx/core/djangoapps/safe_sessions/middleware.py

```diff
@@ -317,13 +317,18 @@ def process_response(self, request, response):
 
         if not _is_cookie_marked_for_deletion(request) and _is_cookie_present(response):
             try:
+                is_from_logout = _is_from_logout(request)
```

Comment:
@nasthagiri Its being skipped [here](https://github.com/edx/edx-platform/commit/7f58b04e373abd779da1ba375b7753cb26fa4081#diff-8013540d784cc3f76f4e59b9c85478f2R364). Setting the logging level is a great idea, I will be doing that! I have to change the specified log to warning level as well.